packages/flutter/test/widgets/focus_scope_test.dart

```diff
@@ -1336,4 +1336,113 @@ void main() {
 
     expect(key.currentState.focusNode.canRequestFocus, isFalse);
   });
+
+  testWidgets('canRequestFocus causes descendants of scope to be skipped.', (WidgetTester tester) async {
```

Comment:
Should there also be a new test for skipTraversal since that logic also changes in this PR?

Reply:
Added a test.  I don't actually need to change the code for `skipTraversal`: it's getter doesn't depend on the enclosing scope, so the code change doesn't change the logic.

It doesn't hurt to have another test though, and I figured that changing the code to match what I did for `canRequestFocus` made it more future proof (and didn't actually change anything).